### PR TITLE
cmd: add go-template option for inspect commands

### PR DESCRIPTION
**Pull request by fuweid**
1. Add github.com/pkg/errors package in go.mod;
2. Add go-template value and template option to
inspect{,p,i}/info/imagesfsinfo subcommands;
3. Add "table" as default value in ps/pods subcommands.

Signed-off-by: Wei Fu <fuweid89@gmail.com>

### CODE REVIEW

File: cmd/crictl/templates.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```
Comment (saschagrunert) [inline cmd/crictl/templates.go]:
```suggestion
Copyright 2020 The Kubernetes Authors.
```

File: cmd/crictl/templates.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"text/template"
+
+	pkgerrors "github.com/pkg/errors"
```
Comment (saschagrunert) [inline cmd/crictl/templates.go]:
```suggestion
	"github.com/pkg/errors"
```

File: cmd/crictl/templates_test.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```
Comment (saschagrunert) [inline cmd/crictl/templates_test.go]:
```suggestion
Copyright 2020 The Kubernetes Authors.
```

File: go.mod [hunk 4]
```diff
@@ -14,6 +14,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/selinux v1.3.1-0.20190929122143-5215b1806f52
 	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
```
Comment (saschagrunert) [inline go.mod]:
```suggestion
	github.com/pkg/errors v0.9.1
```

**Comment by fuweid** [comment]
@saschagrunert thank you for review! Updated. PTAL

**Comment by fuweid** [comment]
Is there anything wrong about `cla/linuxfoundation` check?

**Pull request merged**